example.json

```diff
@@ -22,6 +22,7 @@
             "_type":"document",
             "title":"Button idea",
             "description":"Click here to see a proposed button idea",
+            "filename":"Buttons.pdf",
```

Comment:
I was thinking that filename might actually be translatable, e.g. if you (re)use an image from the asset manager that has an english name in your different lang versions, you could have it downloaded to a filename that is appropriate for the language of the course.